tools/check.py

```diff
@@ -10,37 +10,39 @@
 
 Call at command line with flag -h to see options and usage instructions.
 """
+from __future__ import print_function
 
 import argparse
-import collections
 import glob
 import hashlib
+import itertools
 import logging
 import os
 import re
 import sys
 
-import CommonMark
-import yaml
+import json
+import pypandoc
+import pandocfilters
 
 import validation_helpers as vh
-
+import filters.common as fc
```

Comment:
After installing requirements.txt to Python2, I still get an error that no module named `filters.common` exists. Is this a separate dependency?